src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,89 @@
-import React from 'react';
+import { Comment, CommentData } from '../types/Comment';
+import React, { Dispatch, FormEvent, useState } from 'react';
+
+import { Post } from '../types/Post';
+import { postComment } from '../api/postComment';
+
+interface Props {
+  selectedPost: Post;
+  setComments: Dispatch<React.SetStateAction<Comment[]>>;
+  setIsError: Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Please make prop types more generic

```suggestion
  setComments: (comments: Comment[]) => void;
  setIsError: (isError: boolean) => void;
```